src/esbmc/esbmc_parseoptions.cpp

```diff
@@ -1580,6 +1580,12 @@ bool esbmc_parseoptionst::process_goto_program(
       goto_k_induction(goto_functions, msg);
     }
 
+    if(cmdline.isset("goto-contractor"))
+    {
+      interval_analysis(goto_functions, ns);
+      goto_contractort(goto_functions, msg);
+    }
```

Comment:
this will run the interval analysis twice if the `--interval-analysis` is used with the new flag.

You can change:
```
    if(cmdline.isset("interval-analysis"))
      interval_analysis(goto_functions, ns);
```
to
```
    if(cmdline.isset("interval-analysis") || cmdline.isset("goto-contractor"))
      interval_analysis(goto_functions, ns);
```
and remove the interval analysis from this branch.

Reply:
done.